VTCareService/src/main/java/edu/vt/cs/vtcare/vtcareservice/dao/ProviderDao.java

```diff
@@ -0,0 +1,41 @@
+package edu.vt.cs.vtcare.vtcareservice.dao;
+
+import edu.vt.cs.vtcare.vtcareservice.db.VTCareJDBC;
+import edu.vt.cs.vtcare.vtcareservice.models.Provider;
+
+import java.sql.*;
+
+public class ProviderDao {
+    private Connection connection;
+
+    public ProviderDao() throws Exception {
+        connection = VTCareJDBC.getInstance().getConnection();
+    }
+
+    private static final String CREATE_PROVIDER_SQL =
+            "INSERT INTO providers (provider_name, email, provider_password, gender, date_of_birth, address, contact, specialization, experience)" +
+            "VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)";
```

Comment:
I believe that SQL statements should be taken into stored procedures rather than on code.

Reply:
So again, stored procedures are reserved for SQL statements that are used again and again in the code. And they are used when we need to store a lot of business logic in the stores. We have to use proper features at proper places, can't use them just for the sake of it. Please study the use case for when to have stored procedures in the system. This is not where stored procedures are used.